src/index.tsx

```diff
@@ -1,10 +1,40 @@
-import React from "react";
+import * as React from "react";
```

Comment:
this style of import should ne be needed and indicates following an older react guideline. why was this changed?

Reply:
Similar to the resume import, this is a auto-complete and auto-correction suggestion that went wrong, has since been deleted